README.md

```diff
@@ -3,13 +3,21 @@ Log management tool
 
 ## Requirements 
 
-Install Docker and Node
+Install Docker Desktop, Docker Compose, and Node
```

Comment:
Maybe add something about looking at the Docker Setup section for how to install Docker